infra/main.bicep

```diff
@@ -156,6 +162,9 @@ module monitoring 'core/monitor/monitoring.bicep' = {
 module openAi1RoleApim 'core/security/role.bicep' = {
   scope: resourceGroup
   name: 'openai1-role-apim'
+  dependsOn: [
```

Comment:
not needed, apim is automatically recognized as a dependency because of the param.
There's no dependency on openAi1 as the role is set at RG scope